modules/release-notes/ref-release-notes-technology-preview.adoc

```diff
@@ -78,29 +78,32 @@ The following tables list products and their versions compatible with {product-s
 | Version
 
 | `@backstage/backend-app-api`
-| 1.0.0
+| 1.0.1
+
 
 | `@backstage/backend-defaults`
-| 0.5.0
+| 0.5.2
 
 | `@backstage/backend-dynamic-feature-service`
-| 0.4.1
+| 0.4.4
+
 
 | `@backstage/backend-plugin-api`
-| 1.0.0
+| 1.0.1
 
 | `@backstage/catalog-model`
 | 1.7.0
 
 | `@backstage/cli-node`
-| 0.2.8
+| 0.2.9
 
 | `@backstage/config`
 | 1.2.0
 
 | `@backstage/config-loader`
 | 1.9.1
 |===
+
 .Additional resources
 * link:https://issues.redhat.com/browse/RHIDP-5070[RHIDP-5070]
 

```

Comment:
@jmagak There are new dynamic plugins included in RHDH 1.4, in previous RNs we listed these (see [here](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html-single/release_notes/index#technology-preview-rhidp-3713))

As far as I know, we are adding the following plugins have been added in this release:

- Signals (frontend) - RH Tech Preview
- Notifications (backend) - RH Tech Preview
- Notifications (frontend) - RH Tech Preview
- Notifications Module Email - RH Tech Preview

@reestr @nickboldt @kadel Can you confirm if these should be mentioned in the RNs?